README.md

```diff
@@ -737,6 +737,7 @@ window.addEventListener("MSSDK:redeem-coupon-failed", evt =>
 | `MSSDK:Purchase-loaded`              | `null`                                                                                   | The event will be emitted when Purchase component data is loaded.                                                                                                                                                     |
 | `MSSDK:purchase-successful`          | `{payment: { ...paymentResponse}`                                                        | The event will be emitter after successful Adyen payment action. `paymentResponse` object is just as the repsonse from [Adyen payment response](https://developers.cleeng.com/reference/payment-with-adyen-connector) |
 | `MSSDK:purchase-failed`              | `{reason: "Rejection reason"}`                                                           | The event will be emitter after failed Adyen payment action.                                                                                                                                                          |
+| `MSSDK:Adyen-error`                  | `{error: "Error message", fieldType: "fieldType"}`                                       | The event will be emitted when any of the Adyen errors occur. The information about the error and the fieldType will be available in the event detail. The event will also be emitted when the error disappears after the first successful input verification. The error, in this case, will be an empty string. |
```

Comment:
I will be more specific here, like: 

The event will be emitted when any of the Adyen errors occur, or when the user fixes the input and the error message disappears. 
- error - string with an error message. It will be empty when the error message disappears from the form.
- fieldType - informs for which field error occurred. Possible values: X/Y/Z.